# Astronomical Telescopes and Instruments 2020:
# Exercises on Telescopes
# Due on 13 october 2020 at 14:15
# Auke Bruinsma, s1594443

## 1. Maksutov Corrector
A corrector plate in front of a spherical primary mirror makes a telescope with a very large field of view. One
implementation is the aspherical corrector plate used in a Schmidt telescope. Another version, the Maksutov
design, has a spherical meniscus lens corrector in front of the spherical primary mirror. In this exercise, you
will go through the steps of designing a Maksutov telescope, analyzing its performance and then comparing it
to single-mirror telescopes without a transmitting corrector. **Make sure to include all numerical values
necessary to recreate your TOD design.**

### 1.1
The Maksutov corrector is an achromatic, single lens. Use the expression for the focal length of a thick
lens and require that the focal length is invariant under a change of the index of refraction, i.e. $\frac{\partial f}{\partial n} = 0 $. Derive the relation between the two radii of curvature, $R_1, R_2$ and the thickness $d$ for an achromatic meniscus lens. The radii of curvature of a meniscus lens have the same sign.

**Answer**:

![](fig_4.1.1.jpeg)

### 1.2
The thickness and radii of curvature of the corrector can be further optimized to compensate for the
spherical aberration of the primary mirror. Assume that the corector is made from fused silica ($n=1.46$ at
550 nm). If the primary, spherical mirror has a radius of curvature $R$ and the telescope has an $f$-number of
$2.5$, choose the thickness of the corrector as $d = 0.025|R|$ and $R_1 = 0.2087 R$ to compensate the primary’s
spherical aberration. Calculate $R_2$ using the equation derived above.

**Answer**:

In [7]:
# Initial data.
lam = 550e-9 #m
n = 1.46
f_number = 2.5
d = 0.025 # times absolute value of R
R_1 = 0.2087 # times R

# Now I will apply my derived formula from the previous subquestion.
R_2 = -1 * R_1 * (-1 + d/R_1 * ((n-1)/n * (1 + 1/n)))
print(f'R_2 = {R_2:.5f} R')

R_2 = 0.19543 R


So using the equation above derived, $R_2$ equals $ 0.19543R$.

### 1.3
Place the corrector at a distance of $ 0.2667|R| $ behind the entrance aperture and the mirror at $ 0.8543|R| $
behind the aperture to minimize the aberrations of the complete system. Determine the performance (rms
spot size) of the Maksutov design for field angles of $0$ and $3$ degrees.

**Answer**:

I have chosen R = -100
Following the instructions for the distances for the corrector and the mirror, and using the the results from question 1.2, I have the following data:

In [8]:
import numpy as np
R = -100
d1 = 0.2667 * np.abs(R)
d2 = 0.8543 * np.abs(R)
R1 = 0.2087 * np.abs(R)
R2 = 0.19543 * np.abs(R)
center_thickness = 0.025 * np.abs(R)
print(f'd1 = {d1}')
print(f'd2 = {d2}')
print(f'R1 = {R1}')
print(f'R2 = {R2}')
print(f'center_thickness = {center_thickness}')

d1 = 26.669999999999998
d2 = 85.42999999999999
R1 = 20.87
R2 = 19.543
center_thickness = 2.5


Using the above values, and setting the diameter of the corrector $d = 28.01$, I obtained the following setup:

![](fig_4.1.3.png)

The performance is shown in the table below together with the performances of the single spherical mirror and the Newtonian telescope.

### 1.4
Compare these performance parameters to a single spherical mirror and a Newtonian telescope with the
same $f$-number and focal length.

**Answers**:

| Type | Average rms spot size 0 degrees | Average rms spot size 3 degrees |
|---|---|---|
| Maksutov Corrector | $ 0.23456 $ | $ 0.24556 $ |
| Single spherical mirror | $ 0.01457 $ | $ 0.01570 $ |
| Newtonian telescope | $ 0.01965 $ | $ 0.02835 $ |

The single spherical mirror setup was simply obtained by deleting the corrector, and the newtonian telescope by changing the mirror from spherical to parabolic. As you can see, the Maksutov corrector performs far worse.

## 2. Ritchie-Chretien Telescope

A Cassegrain telescope consists of a parabolic primary and a hyperbolic secondary mirror, which perfectly
reimages the on-axis focus of the primary mirror to a location behind the primary mirror. Such a two-mirror
telescope has no spherical aberration, but it suffers from coma. The Ritchie-Chretien telescope is a modified
Cassegrain telescope design where the off-axis coma of the primary mirror is corrected by making the primary
mirror slightly hyperbolic and the conic constant of the secondary mirror is adjusted such that spherical aber-
ration is still cancelled. **Make sure to include all numerical values necessary to recreate your TOD
design.**

### 2.1
Design a Ritchie-Chretien telescope with a primary mirror diameter of 81.15 mm and a focal length of 144.875 mm and a secondary mirror with a diameter of 11.078 mm and a focal length of 22.669 mm. Determine the conic constants, the separation between the two mirrors, and the back-focal distance using the equations shown in the lecture and provide an image of the design. Hints: use a source beam of 80mm diameter and aim it at the primary mirror.

**Answer**:

![](fig_4.2.1.1.jpeg)

**I am filling in all above equations below in the jupyter notebook.**

In [9]:
# Calculations.
d1 = 8.115e-2 # Primary mirror diameter [m].
d2 = 1.1078e-2 # Secondary mirror diameter [m].
f1 = 1.44875e-1 # Primary mirror focal length [m].
f2 = 2.2669e-2 # Secondary mirror focal length [m].

rho = f2/f1 # Mirror ratio radii curvature.
k = d2/d1 # Ratio ray heights mirror margins.

m = rho/(rho-k) # Transverse magnification secondary.
beta = k*(m+1)-1

K1 = -1 - (2*(1+beta))/(m**2*(m-beta)) # Conic constant 1.
K2 = -((m+1)/(m-1))**2 - (2*m*(m+1))/((m-beta)*(m-1)**3) # Conic constant 2.

W = (1-k)*f1 # Separation between the two mirrors [m].
f1_beta =  f1*(m*k-1) + W # Back focal distance [m].

In [10]:
# Results.
print(f'rho = {rho:.4f}')
print(f'k = {k:.4f}')
print(f'm = {m:.4f}')
print(f'beta = {beta:.4f}')
print(f'K1 = {K1:.4f}')
print(f'K2 = {K2:.4f}')
print(f'W = {W:.4f} m')
print(f'f1_beta = {f1_beta:.4f} m')

rho = 0.1565
k = 0.1365
m = 7.8392
beta = 0.2067
K1 = -1.0051
K2 = -1.7271
W = 0.1251 m
f1_beta = 0.1353 m


So my final results are:

Conic constant $K_1 = -1.0051$

Conic constant $K_2 = -1.7271$

Separation between the two mirrors, $W = 125.1$ mm

Back-focal distance $f_1 \beta = 135.3$ mm


**Implementing all above relevant parameters (focal lengths, mirror diameters, distance beteen the mirros, conic section values, beam diameter) in TOD gives me the following setup**:

![](fig_4.2.1.2.png)

### 2.2 
Show that your design is free of aberrations on-axis and determine the performance for a field angle of 0.5 degrees. What is the major aberration at 0.5 degrees? Hint: increase the diameter of the secondary mirror so that there is no vignetting at a field angle of 0.5 degrees.

**Answer**: I have put the results in the table below. I used all relevant values from question 2.1 to make the setup,so those are the focal lengths, the diameters, the distance between the mirrors, the radii of curvature. To make sure all fields are included, so also those with incoming angle of 0.5 degrees, I slightly made the diameter of both mirrors larger. I have determined the on-axis abberation by putthing the image plane in the focus of just 1 field so it does not take the average of multiple fields.

| Type | On-axis abberation | 0.5 degrees off-axis |
|---|---|---|
| Straight image plane | 0.0000110461 | 0.0271768 |
| Curved image plane | 0.0000110492 | 0.00641560 |

As you can see the value I obtained for the straight image plane for the on-axis abberation is really small, so I conclude this design is free of on-axis abberations.

The setup I used is show in the image below:

![](fig_4.2.2.png)

### 2.3 
Assess the performance that would be achieved with this telescope if a curved image plane would be used. What is the radius of curvature of the image and what is the rms spot diameter at a field angle of 0.5 degrees? Hint: You can change the radius of curvature of the image to a finite value; reduce the image diameter as much as possible.

**Answer**: I changed the radius of curvature of the image plane. First I placed the image plane in the focus of the on-axis beam. Then I empirically derived a value for the radius of curvature, $R = -19.4$. This value makes sures the image plane approximately goes through the center of the other two off-axis foci. I used a diameter of 23. The results can be seen in the table above. As expected the on-axis abberation is of similar size, and the 0.5 degrees of-axis abberation is much lower. The setup can be seen in the image:

![](fig_4.2.3.png)

### 2.4
What is the remaining aberration? Hint: move the image to either side of the best focus position.

**Answer**: The remaining abberation is astigmatism. This can also be seen by zooming-in on the off-axis abberation. There is not one focus point, the rays have different foci. 